components/dial/stories/dial.stories.js

```diff
@@ -1,5 +1,5 @@
 import { Template } from "./template";
-
+import { html } from "lit";
 /**
  * A dial is an input control used for selecting a value within a range, similar to a slider. It's often used in audio and video mixing and editing applications, where horizontal space is limited.
  */
```

Comment:
I didn't migrate the "usage notes" for dial (it was a funny quip about turning it up to 11). I'm happy to though 😆

Reply:
I don't think the `.is-focused` class is doing anything when you toggle the control in storybook. There's nothing in the `index.css` file for dial regarding that class and any styles.